Post templates (#469)
ropensci/roweb2: 2019-06-04-post-template.md

---
slug: "post-template"
title: Wonderful title
package_version: 0.1.0
authors:
  - Author Name
date: 2019-06-04
categories: blog
topicid:
tags:
- Software Peer Review
- R
- community
# delete the line below
# if you have no preferred image
# for Twitter cards
twitterImg: img/blog-images/2019-06-04-post-template/name-of-image.png
---

Everywhere in this template (YAML, paths to images), you should change "post-template" to the slug of your post, and "2019-06-04" to the publication date.

Introduction including outline of the post.

### First awesome section

> I like Hugo[^1]. Yep this was a footnote.

Reviewer: ```suggestion
I like Hugo[^1]. Yes, that is how you add a footnote.
```

Reviewer: I changed the words in case "yep" (which I would use :-) ) is too colloquial for people who are less used to English

#### First awesome subsection of the first awesome section

> Below is an image. We recommend you use a Hugo shortcode to include it.

Reviewer: ```suggestion
Here's how to use a Hugo shortcode to add an image. 
```

changed the wording to match the clear tweet text below

{{< figure src = "/img/blog-images/2019-06-04-post-template/name-of-image.png" width = "200" alt = "this is the alternative text" >}}

#### Second awesome subsection

Here's how to use a Hugo shortcode to embed a tweet. We recommend the use of [Hugo shortcodes](https://gohugo.io/content-management/shortcodes/) to include tweets, Vimeo or Youtube videos, gists, etc.

{{< tweet 1138216112808529920 >}}

### Conclusion

Have fun writing your blog post!

> [^1]: Hugo! https://gohugo.io/

Reviewer: ```suggestion
Here's how to add the footnote text for your reference above.

[^1]: Hugo! https://gohugo.io/
```
